common/FormattingUtils.js

```diff
@@ -68,17 +68,17 @@ const createFormatFunction = (web3, numDisplayedDecimals, minDisplayedPrecision)
     formatWithMaxDecimals(formatWei(valInWei, web3), numDisplayedDecimals, minDisplayedPrecision, false);
 };
 
-// generate a etherscan link prefix
-function createEtherscanLinkFromtx(web3) {
+// generate an etherscan link prefix. If a networkId is provided then the URL will point to this network. Else, assume mainnet.
+function createEtherscanLinkFromtx(networkId) {
   // Construct etherscan link based on network
-  const networkId = web3.networkId;
   let url;
   if (networkUtils[networkId]) {
     url = `${networkUtils[networkId].etherscan}`;
   } else {
     // Default to mainnet, even though it won't work for ganache runs.
     url = "https://etherscan.io/";
   }
+  console.log("URL", url);
```

Comment:
Did you mean to leave this `console.log()`?

Reply:
oversight, thanks for catching this.